pkg/ui/ui.go

```diff
@@ -37,12 +37,20 @@ func Proceed(message string) bool {
 
 func Select(message string, options []string, defaultValue ...string) string {
 	sort.Strings(options)
+	return doSelect(message, options, defaultValue)
+}
+
+func SelectNoSort(message string, options []string, defaultValue ...string) string {
+	return doSelect(message, options, defaultValue)
+}
+
+func doSelect(message string, options []string, defaultValues []string) string {
 	prompt := &survey.Select{
 		Message: message,
 		Options: options,
 	}
-	if len(defaultValue) == 1 {
-		prompt.Default = defaultValue[0]
+	if len(defaultValues) == 1 {
```

Comment:
I kept the name singular because the intent is to fake optional arg using variadic ones… Granted, there should be better documentation than the parameter name :)

Reply:
I only changed it because I needed to change it's type to make `go` happy. Should I change it back?

Reply:
Fixed